### DISCUSSION TITLE
Is it possible to download selected files or folders from the Web Part?

### DISCUSSION THREAD

**Discussion by johnguilbert1970** [microsoft-search/pnp-modern-search#3268] [Q&A]
Hi.
Major requirement is to be able to download multiple folders or files selected in a search. Is this currently possible from the Web part or can you do using custom update?
Help really appreciated as on tight deadline.
Thanks.
John.

**Comment by kasperbolarsen** [comment]
We would very much welcome a PR that would enable users to download the search results, either in a text format like csv or as zip files or similar. 
Currently the web parts does not have that capability.

**Comment by patrikhellgren** [comment]
This will be possible from the next version where #3606 is included.
